.gitignore

```diff
@@ -1,6 +1,7 @@
 .DS_Store
 
 target
+kit/src/compiled_styles.css
```

Comment:
Honestly, its really tempting to have this be outputted to OUT_DIR, similar to whats done #96 and include it from that path instead. Would keep the codebase clean, but that would be done in a separate PR.